XWebView/XWVHttpConnection.m

```diff
@@ -105,7 +105,7 @@ - (NSURL *)rootURL {
     NSURL *root;
     if (_delegate && [_delegate respondsToSelector:@selector(documentRoot)]) {
         root = [NSURL fileURLWithPath:_delegate.documentRoot isDirectory:YES];
-        assert(root);
+        NSCAssert(root != nil, @"<XWV> you must set a valid documentRoot");
```

Comment:
According to [Foundation Functions Reference](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Miscellaneous/Foundation_Functions/#//apple_ref/c/macro/NSAssert), NSCAssert should be used only in C function. In ObjC method, NSAssert is the proper macro.